R/pkg/DESCRIPTION

```diff
@@ -54,5 +54,5 @@ Collate:
     'types.R'
     'utils.R'
     'window.R'
-RoxygenNote: 5.0.1
+RoxygenNote: 6.0.1
```

Comment:
It does it automatically when someone having a newer roxygen2 installed is running the R build.
From looking at that code, there isn't an option to disable the behavior of updating this string automatically. And, I don't think there is a way to fix the version either, other than mucking with R packages that are installed.